src/VstsDemoBuilder/Templates/AzureSentinel/ServiceEndpoints/Azure Sentinel Template-code.json

```diff
@@ -1,16 +1,16 @@
 {
-  "data": {
-
-  },
+  "data": {},
   "name": "Azure Sentinel Template-code",
   "type": "git",
   "url": "https://dev.azure.com/vandyr/Azure Sentinel Template/_git/Azure Sentinel Template",
   "authorization": {
     "scheme": "UsernamePassword",
     "parameters": {
       "username": "$username$",
-      "password": "$password$"
+      "password": "$username$"
```

Comment:
Is this correct?  Using $username$ for the password?

Reply:
I'm sorry, didn't notice while copying 🙂

Reply:
Still looks like this says `$username$`!